### ISSUE TITLE
Minimum esx license

### ISSUE THREAD

**Issue by dewiestr** [cuckoobox/cuckoo#416]
Hi, 

I recently found out (the hard way) that the ESX Hypervisor ( free ) version does not support the vSphere API, which Cuckoo uses as interface. Does anyone know ( maybe this can be inserted into the documentation) what the minimum license version of vSphere is ? And Maybe we could switch to a custom script ( not via libvirt) that uses SDK ( which is supported by the free version) ? 

Regards

**Comment by jekil** [comment]
@KillerInstinct already answered, thanks.
I added a general paragraph about this to the documentation, but for us it is hard to follow VMware marketing and license plans changes, so I added only a generic statement.

**Comment by dewiestr** [comment]
After a couple of ping-pong mails to the vmware support, I finally found out which license is needed. Seeing that, all you need for cuckoo is the vSphere API, you should have one of the following versions:

http://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=2001113 <http://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=2001113>

Regards,
